### DISCUSSION TITLE
How do i install it?

### DISCUSSION THREAD

**Discussion by DaviMeurer** [Reloaded-Project/Reloaded-II#358] [Q&A]
I know it's kind of a dumb question, but how do i install it properly?

**Comment by Sewer56** [comment]
Can you give me the specific game you want to install with?

Apologies for late response, I forgot to enable notifications for myself.

**Comment by RobLi2211** [comment]
Hi,
it's not clear to me how I download the files as a user. Downloading the code does not seem to be an installable version with an exe file. Can you please help?
Thanks

**Comment by Sewer56** [comment]
There's a [Setup.exe](https://github.com/Reloaded-Project/Reloaded-II/releases/download/1.27.12/Setup.exe) on the [Releases](https://github.com/Reloaded-Project/Reloaded-II/releases/tag/1.27.12) page.

I do think the button could be more discoverable however.

**Reply by RobLi2211** [reply]
Ah! You have to scroll to the very bottom of the page. Thanks a lot for the fast reply :)
